plugins/draw-tools.js

```diff
@@ -584,8 +584,8 @@ window.plugin.drawTools.snapToPortals = function() {
       if (visibleBounds.contains(ll)) {
         testCount++;
         var newll = findClosestPortalLatLng(ll);
-        if (!newll.equals(ll)) {
-          layer.setLatLng(new L.LatLng(newll.lat,newll.lng));
+        if (newll.lat !== ll.lat || newll.lng !== ll.lng) {
```

Comment:
```suggestion
        if (newll.lat !== ll.lat || newll.lng !== ll.lng) { // must be strict
 ```

I suppose such short comment would be enough.

Reply:
Ok